x/auth/ante/ante_test.go

```diff
@@ -480,7 +480,7 @@ func (suite *AnteTestSuite) TestAnteHandlerFees() {
 		{
 			"signer as enough funds, should pass",
 			func() {
-				accNums = []uint64{7}
+				accNums = []uint64{6}
```

Comment:
How about changing this account number like cosmos-sdk?
https://github.com/cosmos/cosmos-sdk/blob/47f46643affd7ec7978329c42bac47275ac7e1cc/x/auth/ante/ante_test.go#L485